students/hangu/users/views.py

```diff
@@ -52,13 +52,15 @@ def post(self, request):
             user_email = user_data['user_email']
             password   = user_data['password']
 
-            if not User.objects.filter(user_email = user_email).exists():
-                return JsonResponse({"message": "이메일이 틀렸습니다"}, status=401)
+            user = User.objects.filter(user_email = user_email)
```

Comment:
`get()` 활용해서 전체적으로 코드를 수정해보시겠어요?
단 한번의 쿼리만으로 해결할 수 있어보입니다.
```suggestion
            user = User.objects.get(user_email = user_email)
```